src/main/java/wolox/training/models/Book.java

```diff
@@ -116,6 +125,7 @@ public String getYear() {
     }
 
     public void setYear(String year) {
+        checkArgument(!Strings.isNullOrEmpty(year));
         this.year = year;
```

Comment:
After checking if it is null or empty, you can check if the content of the year field is a number.

You can use the StringUtils.isNumeric method. [import org.apache.commons.lang3.StringUtils;]